### PR TITLE
Switched to new design

### PR DESCRIPTION
**Pull request by moshe742**

### CODE REVIEW

**Comment by tomer** [comment]
I've checked out a local copy of this branch, but it seems that there has no style at all.

File: en/index.html [hunk 1]
```diff
@@ -1,108 +1,103 @@
...
-      <div>
-        Will open soon
-      </div>
-
-      <a id="schedule" name="schedule"></a>
-      <div id="schedule-section">
-        <h2>Agenda</h2>
-        <div>
-          Final agenda will be published in July. Want to participate? The call for presentations is open! Please fill in the form below:
+<html class="no-js" lang="en">
```
Comment (tomer) [inline en/index.html:2]: Please DO NOT override the whole file when you are doing just minor changes. This breaks the blame view.

**Comment by tbrisker** [comment]
@moshe742 This looks like practically no style at all has been applied, see screenshot:
screenshot from 2016-06-12 20-06-38
If this is what it is intended to look like, a big :-1: from me, the current design may be dated but still looks better then nothing at all.
There are also merge conflicts, please resolve them (rebase on ap2016, handle any conflicts, and recommit)

**Comment by tbrisker** [comment]
Also, please don't comment out entire files, if a file needs removing remove it.

**Comment by moshe742** [comment]
I think we should talk about what exactly we want from the new design, since this is based on foundation (modern css framework).
Should I remove the files and resolve the conflicts or wait for the decision on the design before working?

**Comment by tbrisker** [comment]
@moshe742 a framework is just that - a framework. It gives you a nice frame to build upon but nothing more then that. I'm in favor of using a modern framework that allows, for example, easy handling of mobile devices, but without an actual design it doesn't look much better then a text file. I'd say that you should only continue working on this PR if you plan to add a design on top of the framework, otherwise feel free to close it.

**Pull request closed without merging**